src/app/hero/home/home-container/home-container.component.html

```diff
@@ -3,16 +3,42 @@
   <div class="funtionalities">
     <div class="search-bar">
       <input type="text" placeholder="  Search... " #searchedString [(ngModel)]="inputValue">
-      <button type="button" (click)="searchHeroes(searchedString.value)">GO</button>
+      <button type="button" class="header-button" (click)="searchHeroes(searchedString.value)">GO</button>
     </div>
     <div class="sort-button" (click)="sortHeroes()">
-      <button type="button">SORT</button>
+      <button type="button" class="header-button">SORT</button>
     </div>
+    <a (click)="buttonClicked()" class="addIcon">
+      <mat-icon class="icon-color">add_circle</mat-icon>
+    </a>
+  </div>
+  <div id="form-container" *ngIf="addIconClicked">
+    <form [formGroup]="heroForm" (ngSubmit)="onSubmit()">
+      <div class="form-field" [ngClass]="{'has-errors': displayHeroIdErrorMessage}">
+        <label for="heroId">Hero Id&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</label>
+        <input type="number" id="heroId" formControlName="heroId">
```

Comment:
also id need not be an input, it could be hardcoded based on how many results are there